.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think these `.vs` files should be kept in the .gitignore. Same with the 2 files below.